R/extract.R

```diff
@@ -4438,15 +4438,16 @@ setMethod("extract", signature = className("vglm", "VGAM"),
     definition = extract.vglm)
 
 
-# extension for weibreg objects (eha package)
-extract.weibreg <- function(model, include.loglik = TRUE, include.lr = TRUE, 
-    include.nobs = TRUE, include.events = TRUE, include.trisk = TRUE, ...) {
-  
+# extension for weibreg objects (aa package)
+extract.weibreg <- function(model, include.loglik = TRUE, include.lr = TRUE,
+    include.nobs = TRUE, include.events = TRUE, include.trisk = TRUE,
+    include.aic = TRUE, ...) {
```

Comment:
consider changing the order of the arguments in line with other `extract` methods